linux/ibus-keyman/tests/scripts/test-helper.sh

```diff
@@ -9,3 +9,83 @@ function can_run_wayland() {
     return 0
   fi
 }
+
+function generate_kmpjson() {
+  local TESTDIR
+  TESTDIR="$1"
+  pushd "$TESTDIR" > /dev/null || exit
+  KMPFILE="${TESTDIR}/kmp.json"
+  THISYEAR=$(date "+%Y")
+  cat <<-EOF > "$KMPFILE"
+  {
+    "system": {
+      "keymanDeveloperVersion": "10.0.1099.0",
+      "fileVersion": "7.0"
+    },
+    "options": {
+      "readmeFile": "kmx.cpp"
```

Comment:
Technically this is an invalid kmp.json because kmx.cpp isn't listed in the `files` element. `readmeFile` can probably be omitted.

Reply:
Done in #8732.